File: AyoConvertorUI/src/app/models/conversion-request.ts
import { ConversionMethod } from "./conversion-method";

export class ConversionFormData{

    conversionMethod?: ConversionMethod;
    inputValue?: string;

  
}

export class ConversionRequest{

    conversionMethodCode?: string;
    inputValue?: number;
    
    constructor(_code : string, _inputValue: number)
    {
     this.conversionMethodCode = _code;
     this.inputValue = _inputValue;
    }
  
}

File: Ayo.ConveterApp/Ayo.ConveterApp.Api/Models/Requests/ConversionRequest.cs
using Ayo.ConverterApp.Core.Models.Conversion;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using CoreModels = Ayo.ConverterApp.Core.Models.Conversion;

namespace Ayo.ConveterApp.Api.Models.Requests
{
    public class ConversionRequest 
    {
       public string ConversionMethodCode { set; get; }
       public double InputValue { set; get; }
    }
}
